### PR TITLE
Better due date

### PR DESCRIPTION
**Pull request by marcobischoff**
Added a dropdown which allows to select +5, +10, +30 days of the current date. You are still able to select a custom date from the date picker. #129

### CODE REVIEW

**Comment by marcobischoff** [comment]
Just a short update for the lint parser

**Comment by hql287** [comment]
This seems to be working but I would like to change the UI a little bit. Can you allow me to push to your fork?

**Comment by marcobischoff** [comment]
I've added you as contributor to my fork.

**Comment by hql287** [comment]
@marcobischoff: Alright, so I tried your fork and it seems to be working as expected. While the code looks short, it poses some issues: 

* The UI looks a bit confusing as there are 2 inputs at the moment. 
* You can't show the payment term on the invoice as the current approach would convert that to a specific date. 
* Code organization 

I think I would solve these issues as follow: 

* The `dueDate` field should be implemented similarly to the `Recipient` field, which has radio buttons letting users know that they can choose a pre-defined payment term (from the dropdown list) or they can pick a custom date (use the current date picker)
* The `dueDate` state would need to keep track of both cases. This would make things easier when it comes to editing the invoice or showing the date/term on the templates.
* I would refactor the current `DueDate` component as `DueDatePicker`, create a new component call `DueDateTerms` to handle the dropdown list and then a wrapper component called `DueDate` and this is where the `dueDate`'s state is kept and updated. 

I would also extract and abstract the calculation (payment term -> specific date) as a helper as well.

File: i18n/en/form.json [hunk 1]
```diff
@@ -32,7 +32,10 @@
     },
     "dueDate": {
       "name": "Due Date",
-      "placeHolder": "Select a Date"
+      "placeHolder": "Select a custom date",
+      "addSomeDaysToCurrentDate": "Add some days to the current date",
+      "addDaysToCurrentDate" : " - Payment seven days after invoice date",
```
Comment (hql287) [inline i18n/en/form.json]: This actually doesn't make sense

**Comment by hql287** [comment]
I was thinking about pushing directly to your fork but I think it would be better to create a new PR so you can compare the 2 approaches easier. 

(Fixing the tests at the moment, will publish it asap.)

File: i18n/en/form.json (hunk 1 above)
Comment (marcobischoff) [reply i18n/en/form.json]: Do you mean line number 37? I thought, I've updated this string to something meaningful.

**Comment by marcobischoff** [comment]
@hql287 sounds a little bit complicated to me, I can give it a try.

File: i18n/en/form.json (hunk 1 above)
Comment (hql287) [reply i18n/en/form.json]: Yes, on line 37, the key and the value don't match.

**Comment by hql287** [comment]
> @hql287 sounds a little bit complicated to me, I can give it a try.

I will create a new PR in a minute then you see what I meant.

**Comment by hql287** [comment]
@marcobischoff: Please take a look at https://github.com/hql287/Manta/pull/224. 
Let me know what you think. Thanks!

**Pull request closed without merging**